lib/tasks/search.rake

```diff
@@ -0,0 +1,24 @@
+require 'elasticsearch/rails/ha/tasks'
+
+namespace :search do
+  desc 're-index all records, in parallel'
+  task index: :environment do
```

Comment:
It's possible that the /admin button to re-index should use the stage+promote tasks in order to provide zero-downtime index rebuilds. But that can be an optimization later.